.changeset/witty-hairs-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nacelle/shopify-cart': patch
+---
+
+feat: depaginate cart line items
```

Comment:
```suggestion
Added pagination logic for cart line items. The `{ cart }` response will now contain all `cart.lines`, rather than just the first page of `cart.lines` results.
```